openreview/arr/arr.py

```diff
@@ -658,17 +655,18 @@ def add_readers_to_arr_submission(submission):
 
         commitment_submissions = client.get_all_notes(invitation=submission_id)
 
-        count=0
-        for note in commitment_submissions:
+        def process_commitment_submission(note):
             arr_submission_link = note.content.get('paper_link', {}).get('value')
             if arr_submission_link:
                 arr_submission_id = arr_submission_link.split('=')[-1]
                 arr_submission = openreview.tools.get_note(client, arr_submission_id)
                 if arr_submission and 'aclweb.org/ACL/ARR/' in arr_submission.invitations[0]:
-                    print('API 2 submission found', note.id, note.number, arr_submission.id, arr_submission.number, arr_submission.invitations[0])
                     create_readers_group(arr_submission, note)
                     add_readers_to_arr_submission(arr_submission)
-                    count+=1
+                    return True
+            return False
 
+        result = openreview.tools.concurrent_requests(process_commitment_submission, commitment_submissions, desc='process_commitment_submissions')
+        count = sum(1 for r in result if r)
```

Comment:
Could you use len instead?
```suggestion
        count = len(result)
```